### DISCUSSION TITLE
Website ra3xdh.github.io update

### DISCUSSION THREAD

**Discussion by ra3xdh** [ra3xdh/qucs_s#834] [Announcements]
I have updated Qucs-S webpage https://ra3xdh.github.io/ and replaced some outdated screenshots, guides, infos etc. Please check.

**Comment by tomhajjar** [comment]
For Windows, will people know to create the  "C:\Users\xxx\.qucs - S" project directory that is created only if you use the Windows installer?

**Comment by ra3xdh** [comment]
The Windows installer doesn't create any subdirectories in user home directory. It is created on the first application start. The default project directory has changed to  `$HOME/QucsWorksapce` after #145 The Qucs-S doesn't use `$HOME/.qucs` for projects and cache anymore. Old settings are preserved.
